mscore/plugin/api/cursor.h

```diff
@@ -69,24 +69,24 @@ class Cursor : public QObject {
       Q_PROPERTY(int filter     READ filter    WRITE setFilter)
 
       /** MIDI tick position, read only */
-      Q_PROPERTY(int tick         READ tick) // FIXME: fraction transition
+      Q_PROPERTY(int tick         READ tick CONSTANT) // FIXME: fraction transition
```

Comment:
Most of properties in plugin API are not constant, it would be incorrect to add `CONSTANT` attribute to all of them.

Reply:
Hmm, wouldn't all then don't have a WRITE method be CONSTANT?

Reply:
Which would, which would need a NOTIFY method?

Reply:
Need to revert them all?

Reply:
According to the documentation [here](https://doc.qt.io/qt-5/properties.html), `CONSTANT` attribute means that reading this property for the same object will always return the same value. As most of objects in plugin API just expose properties of the underlying libmscore objects, and those objects may change its state while score is being edited (by user or by plugin), such properties are not `CONSTANT` in the sense defined in the documentation.

Having `NOTIFY` signals for those properties would be ideal, but that would require adding a mechanism that would allow objects in libmscore report their properties changes. That would be a non-trivial and potentially resource-consuming change (as some properties like positions of elements change for many objects after every layout), and the benefit of this is unclear. `NOTIFY` signals would enable those properties being used in property bindings, but this seems to be not quite useful for plugin API objects as those are not normally used as usual QML components but are rather being handled in JavaScript code of functions or signal handlers.

For those objects that are intended to be used in QML markup (like [`PluginAPI`](https://github.com/musescore/MuseScore/blob/3.x/mscore/plugin/api/qmlpluginapi.h) object which is a root component of any plugin or objects from [`util`](https://github.com/musescore/MuseScore/blob/3.x/mscore/plugin/api/util.h) part of the API) it would indeed be good to mark those properties that do not change as `CONSTANT` and possibly add `NOTIFY` signals for other properties. However I doubt that adding those notifier signals to objects that cannot be created directly in QML code worth the effort.

Reply:
OK, guess I'll revert that part of the changes then